tests/serialisation_tests/test_serialisation.py

```diff
@@ -91,12 +99,14 @@ def test_serialise_download_data_specific_tab(seeded_test_client, additional_tes
     """Test that serialiser func doesn't return all "sheets" of data if "sheets_required" passed as param."""
     base_query = download_data_base_query()
     test_serialised_data = serialise_download_data(base_query, sheets_required=["ProgrammeRef"])
-    assert test_serialised_data.keys() == {"ProgrammeRef"}
+    for x, y in test_serialised_data:
+        print(2)
+    # assert test_serialised_data.keys() == {"ProgrammeRef"}
```

Comment:
Is this to just test iterating though the generator? Do we need to print 2s?

Reply:
haha, no that's just so I can stop the debugger, forgot to remove